plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/IDeployImagePageModel.java

```diff
@@ -94,7 +94,17 @@
 	 * @param imageName the name of the Docker Image to use
 	 */
 	void setImageName(String imageName);
-	
+
+	/**
+	 * Since method setImageName(String) ignores attempts to set an empty value, 
+	 * ui gets not synchronized with model as soon as image name is cleared.
```

Comment:
Correct, and besides, the setImageName(String) checks the value for null, and does nothing if so.